packages/loot-core/src/client/actions/account.ts

```diff
@@ -106,7 +106,12 @@ export function syncAccounts(id) {
     }
 
     const { errors, newTransactions, matchedTransactions, updatedAccounts } =
-      await send('nordigen-accounts-sync', { id });
+      await send<{
```

Comment:
This `send` API would need a much better structure to become "type safe", but that will be a solid undertake on its own, so for now we just require some types at call site.

Reply:
Would it be possible to declare a type like...

```ts
interface Handlers {
  'nordigen-accounts-sync': ({ id }) => Promise<void>;
  // TODO: remove once all handlers are typed
  [key: string]: (...args: unknown[]) => Promise<unknown>;
}
```

Reply:
Super nice, thanks for making that so through!